homework2/homework2.js

```diff
@@ -0,0 +1,98 @@
+/*1-ый блок
```

Comment:
Why is everything commented out?

Reply:
i forgot uncomment before pushing. Next time everything will be ok.